core/src/main/java/org/acegisecurity/providers/AuthenticationProvider.java

```diff
@@ -1,7 +1,7 @@
 /*
  * The MIT License
  *
- * Copyright (c) 2004-2010, Oracle Corporation
+ * Copyright 2020 CloudBees, Inc.
```

Comment:
should be in addition not removing (based on my superb up to date legal skilz)
or github is lying that this is a rename

Reply:
Git’s file rename detection is confused. Since, unlike say Mercurial, this happens at view time not commit time, I cannot force it to understand.